<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>management_page</class>
 <widget class="QMainWindow" name="management_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>311</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>车辆管理</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="font">
       <font>
        <family>楷体</family>
       </font>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="car">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QToolBox" name="toolBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>350</width>
             <height>0</height>
            </size>
           </property>
           <property name="currentIndex">
            <number>2</number>
           </property>
           <widget class="QWidget" name="page">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>350</width>
              <height>375</height>
             </rect>
            </property>
            <attribute name="label">
             <string>出售车辆</string>
            </attribute>
            <widget class="QLabel" name="label_2">
             <property name="geometry">
              <rect>
               <x>60</x>
               <y>10</y>
               <width>61</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>楷体</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>厂家:</string>
             </property>
            </widget>
            <widget class="QComboBox" name="comboBox_factory">
             <property name="geometry">
              <rect>
               <x>116</x>
               <y>10</y>
               <width>111</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>楷体</family>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="editable">
              <bool>false</bool>
             </property>
            </widget>
            <widget class="QLabel" name="label_4">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>130</y>
               <width>71</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>楷体</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>报价：</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="lineEdit_price">
             <property name="geometry">
              <rect>
               <x>80</x>
               <y>130</y>
               <width>191</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>楷体</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
            <widget class="QLabel" name="label_5">
             <property name="geometry">
              <rect>
               <x>280</x>
               <y>130</y>
               <width>31</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>楷体</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>万</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_6">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>200</y>
               <width>71</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>楷体</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>数量:</string>
             </property>
            </widget>
            <widget class="QSpinBox" name="spinBox">
             <property name="geometry">
              <rect>
               <x>80</x>
               <y>200</y>
               <width>111</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>楷体</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="frame">
              <bool>false</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
            <widget class="QLabel" name="label_7">
             <property name="geometry">
              <rect>
               <x>200</x>
               <y>200</y>
               <width>111</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>楷体</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>剩余数量：</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_last">
             <property name="geometry">
              <rect>
               <x>290</x>
               <y>200</y>
               <width>51</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>楷体</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>20</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_16">
             <property name="geometry">
              <rect>
               <x>280</x>
               <y>260</y>
               <width>51</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>楷体</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>万</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="lineEdit_total">
             <property name="geometry">
              <rect>
               <x>80</x>
               <y>260</y>
               <width>191</width>
               <height>31</height>
              </rect>
             </property>
            </widget>
            <widget class="QLabel" name="label_17">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>260</y>
               <width>71</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>楷体</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>金额：</string>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton_confirm">
             <property name="geometry">
              <rect>
               <x>41</x>
               <y>334</y>
               <width>111</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>楷体</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>确认</string>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton_cancel">
             <property name="geometry">
              <rect>
               <x>206</x>
               <y>334</y>
               <width>111</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>楷体</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>取消</string>
             </property>
            </widget>
            <widget class="QComboBox" name="comboBox_brand">
             <property name="geometry">
              <rect>
               <x>116</x>
               <y>70</y>
               <width>111</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>楷体</family>
              </font>
             </property>
             <property name="editable">
              <bool>false</bool>
             </property>
            </widget>
            <widget class="QLabel" name="label_3">
             <property name="geometry">
              <rect>
               <x>60</x>
               <y>60</y>
               <width>91</width>
               <height>51</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>楷体</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>品牌:</string>
             </property>
            </widget>
           </widget>
           <widget class="QWidget" name="page_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>350</width>
              <height>375</height>
             </rect>
            </property>
            <attribute name="label">
             <string>新车入库</string>
            </attribute>
            <widget class="QLabel" name="label_8">
             <property name="geometry">
              <rect>
               <x>60</x>
               <y>10</y>
               <width>111</width>
               <height>51</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>厂家：</string>
             </property>
            </widget>
            <widget class="QComboBox" name="comboBox_factory_new">
             <property name="geometry">
              <rect>
               <x>120</x>
               <y>20</y>
               <width>121</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>9</pointsize>
              </font>
             </property>
            </widget>
            <widget class="QLabel" name="label_9">
             <property name="geometry">
              <rect>
               <x>60</x>
               <y>70</y>
               <width>91</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>品牌：</string>
             </property>
            </widget>
            <widget class="QComboBox" name="comboBox_brand_new">
             <property name="geometry">
              <rect>
               <x>120</x>
               <y>71</y>
               <width>121</width>
               <height>31</height>
              </rect>
             </property>
            </widget>
            <widget class="QLabel" name="label_10">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>130</y>
               <width>341</width>
               <height>61</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>请选择新添车辆的数目：(最大值999)</string>
             </property>
            </widget>
            <widget class="QSpinBox" name="spinBox_new">
             <property name="geometry">
              <rect>
               <x>100</x>
               <y>200</y>
               <width>141</width>
               <height>41</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="frame">
              <bool>false</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="accelerated">
              <bool>true</bool>
             </property>
             <property name="maximum">
              <number>999</number>
             </property>
            </widget>
            <widget class="QPushButton" name="btn_confirm_new">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>300</y>
               <width>111</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>确认</string>
             </property>
            </widget>
            <widget class="QPushButton" name="btn_cancel_new">
             <property name="geometry">
              <rect>
               <x>190</x>
               <y>300</y>
               <width>111</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>取消</string>
             </property>
            </widget>
           </widget>
           <widget class="QWidget" name="page_3">
            <attribute name="label">
             <string>新款上市</string>
            </attribute>
            <widget class="QLabel" name="label_11">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>60</y>
               <width>111</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>请输入厂家：</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_12">
             <property name="geometry">
              <rect>
               <x>80</x>
               <y>10</y>
               <width>191</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>新款车辆信息采集</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="edit_factory_newK">
             <property name="geometry">
              <rect>
               <x>120</x>
               <y>60</y>
               <width>181</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
            </widget>
            <widget class="QLabel" name="label_13">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>110</y>
               <width>101</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>请输入品牌：</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="edit_brand_newK">
             <property name="geometry">
              <rect>
               <x>120</x>
               <y>110</y>
               <width>181</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
            </widget>
            <widget class="QLabel" name="label_14">
             <property name="geometry">
              <rect>
               <x>11</x>
               <y>164</y>
               <width>101</width>
               <height>21</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>请输入报价：</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="edit_price_newK">
             <property name="geometry">
              <rect>
               <x>120</x>
               <y>160</y>
               <width>181</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
            </widget>
            <widget class="QLabel" name="label_15">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>200</y>
               <width>111</width>
               <height>51</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>请输入数量：</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="edit_num_newK">
             <property name="geometry">
              <rect>
               <x>120</x>
               <y>210</y>
               <width>181</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
            </widget>
            <widget class="QPushButton" name="btn_confirm_newK">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>290</y>
               <width>111</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>确认</string>
             </property>
            </widget>
            <widget class="QPushButton" name="btn_cancel_newK">
             <property name="geometry">
              <rect>
               <x>180</x>
               <y>290</y>
               <width>111</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>取消</string>
             </property>
            </widget>
           </widget>
          </widget>
          <widget class="QTextEdit" name="textEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'楷体'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;日销售清单：&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:18pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="calc"/>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>311</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>销售管理</string>
    </property>
    <addaction name="actionCar"/>
    <addaction name="actionCalc"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCar">
   <property name="text">
    <string>车辆管理</string>
   </property>
  </action>
  <action name="actionCalc">
   <property name="text">
    <string>销售统计</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
